package.json

```diff
@@ -39,19 +39,19 @@
     "typescript": "^2.6.2"
   },
   "dependencies": {
-    "@types/bluebird": "3.5.18",
+    "@types/bluebird": "3.5.20",
     "@types/dockerode": "2.5.5",
-    "@types/lodash": "^4.14.87",
-    "@types/tar-stream": "^1.6.0",
+    "@types/lodash": "4.14.112",
+    "@types/tar-stream": "1.6.0",
     "ajv": "^6.6.1",
-    "bluebird": "^3.5.1",
-    "docker-progress": "^3.0.1",
+    "bluebird": "^3.5.3",
+    "docker-progress": "^3.0.4",
     "docker-toolbelt": "^3.3.7",
     "dockerode": "^2.5.5",
     "lodash": "^4.17.4",
     "resin-bundle-resolve": "^3.0.1",
-    "resin-compose-parse": "^2.0.1",
-    "resin-docker-build": "^1.0.0",
+    "resin-compose-parse": "^2.0.3",
+    "resin-docker-build": "^1.0.1",
```

Comment:
`resin-docker-build` is the important change here, to incorporate this PR: https://github.com/balena-io/resin-docker-build/pull/31
The other packages were upgraded to match `resin-builder`.